podcasts/Notifications/NotificationsCoordinator.swift

```diff
@@ -0,0 +1,119 @@
+import Foundation
+
+enum NotificationType: String {
+
+    case onboardingSignUp
+    case onboardingImport
+    case onboardingThemes
+    case onboardingUpNext
+    case onboardingFilters
+    case onboardingUpsell
+
+    var title: String {
+        switch self {
+        case .onboardingSignUp:
+            return L10n.notificationsOnboardingSignupTitle
+        case .onboardingImport:
+            return L10n.notificationsOnboardingImportTitle
+        case .onboardingThemes:
+            return L10n.notificationsOnboardingThemesTitle
+        case .onboardingUpNext:
+            return L10n.notificationsOnboardingUpnextTitle
+        case .onboardingFilters:
+            return L10n.notificationsOnboardingFiltersTitle
+        case .onboardingUpsell:
+            return L10n.notificationsOnboardingUpsellTitle
+        }
+    }
+
+    var body: String {
+        switch self {
+        case .onboardingSignUp:
+            return L10n.notificationsOnboardingSignupBody
+        case .onboardingImport:
+            return L10n.notificationsOnboardingImportBody
+        case .onboardingThemes:
+            return L10n.notificationsOnboardingThemesBody
+        case .onboardingUpNext:
+            return L10n.notificationsOnboardingUpnextBody
+        case .onboardingFilters:
+            return L10n.notificationsOnboardingFiltersBody
+        case .onboardingUpsell:
+            return L10n.notificationsOnboardingUpsellBody
+        }
+    }
+
+    var identifier: String {
+        return self.rawValue
+    }
+
+    var link: String {
+        switch self {
+        case .onboardingSignUp:
+            return "pktc://signup"
+        case .onboardingImport:
+            return "pktc://settings/import"
+        case .onboardingThemes:
+            return "pktc://settings/themes"
+        case .onboardingUpNext:
+            return "pktc://upnext/?location=tab"
+        case .onboardingFilters:
+            return "pktc://filters"
+        case .onboardingUpsell:
+            return "pktc://upsell"
+        }
+    }
+}
+
+class NotificationsCoordinator {
+
+    static var shared: NotificationsCoordinator = NotificationsCoordinator()
```

Comment:
⛏️  Can this be a `static let` instead of `static var`?

Reply:
Updated here: [3b3de32](https://github.com/Automattic/pocket-casts-ios/pull/2943/commits/3b3de32011db8803527352ebd94b8d1b92523f07)